ui/desktop/README.md

```diff
@@ -1,6 +1,6 @@
-# Client Connect UI
+# Desktop client UI
```

Comment:
I changed this because I feel Client connect UI might be an initial name, but at this point I think is more accurate to refer as Desktop client UI.